src/main/java/de/tum/cit/aet/artemis/quiz/domain/QuizQuestion.java

```diff
@@ -69,7 +69,7 @@ public abstract class QuizQuestion extends DomainObject {
 
     @Column(name = "points")
     @JsonView(QuizView.Before.class)
-    private Integer points;
+    private Double points;
```

Comment:
Can the points actually be `null`, or would this be a good opportunity to change to a primitive type?
```suggestion
    private double points;
```

Reply:
I agree, `double` would be better here